php/class-customize-snapshot-manager.php

```diff
@@ -745,6 +746,21 @@ public function enqueue_frontend_scripts() {
 	}
 
 	/**
+	 * Enqueue admin scripts.
+	 *
+	 * These files control the behavior and styling of links to remove settings.
+	 * Published snapshots can't be edited, so these files are not needed on those pages.
+	 */
+	public function enqueue_admin_scripts() {
+		global $post;
+		$handle = 'customize-snapshots-admin';
+		if ( isset( $post->post_type ) && ( Post_Type::SLUG === $post->post_type ) && ( 'publish' !== $post->post_status ) ) {
```

Comment:
Hi @weston, it looks like checking the `$hook` argument in this [admin_enqueue_scripts](https://codex.wordpress.org/Plugin_API/Action_Reference/admin_enqueue_scripts) callback should be enough.
This [commit](https://github.com/xwp/wp-customize-snapshots/pull/84/commits/68fa5ae929700ab44b31f171d69d09f97074610b) adds an expression to the conditional:
```(  'post.php' === $hook )```
The [WP_Screen object](https://codex.wordpress.org/Class_Reference/WP_Screen) that `get_current_screen()` returns has some useful properties, including:
```
id = "custom_snapshot",
base = "post",
post_type = "customize_snapshot"
```
But the conditional already checks for most of these:
```if ( ( 'post.php' === $hook ) && isset( $post->post_type ) && ( Post_Type::SLUG === $post->post_type ) && ( 'publish' !== $post->post_status ) ) {```